kde-plasma/plasma-pa/plasma-pa-5.27.7-r1.ebuild

```diff
@@ -31,6 +31,5 @@ DEPEND="
 RDEPEND="${DEPEND}
 	>=dev-qt/qtquickcontrols2-${QTMIN}:5
 	>=kde-frameworks/kirigami-${KFMIN}:5
-	x11-themes/sound-theme-freedesktop
```

Comment:
I'm not against making non-plasma-meta users' life more miserable, but this bit is going against upstream's wishes:
```
find_package(SoundThemeFreedesktop)
set_package_properties(SoundThemeFreedesktop PROPERTIES
    DESCRIPTION "The standard freedesktop sound theme"
    PURPOSE "Required for volume feedback sounds"
    URL "https://www.freedesktop.org/wiki/Specifications/sound-theme-spec/"
    TYPE RUNTIME
)
```